src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
It is because the type is unspecified. It can be a string, int, object or a dictionary. This is just used for text formatting and requires jobject actually.

Example:

```
    {
      "type": "SH01",
      "date": "2013-05-22",
      "description_values": {
        "date": "2013-05-10",
        "capital": [
          {
            "currency": "GBP",
            "figure": "1,001"
          }
        ]
      },
      "action_date": "2013-05-10",
      "description": "capital-allotment-shares",
      "category": "capital",
      "links": {
        "document_metadata": "https://document-api.companieshouse.gov.uk/document/d_H6ycE78OHEUpAwfx1tsHPtk14JpEqAzvYHtGff5NQ",
        "self": "/company/03977902/filing-history/MzA3ODQ0MTU5NGFkaXF6a2N4"
      },
      "pages": 3,
      "barcode": "X28W444H",
      "transaction_id": "MzA3ODQ0MTU5NGFkaXF6a2N4"
    }
```